### PR TITLE
update some desc and some configuration

### PR DESCRIPTION
**Pull request by glkappe**
！！！！！To reduce the complexity of tidb-lightning configuration parameters, please conduct a rigorous technical review！！！！！

该文档的内容更新设置的技术点：
1. tidb-lightning 是从哪个版本开始将 status-port 作为可选项，当后端模式为 "tidb" 时，这决定了文档 merge 到哪些版本

更新原则：
1.  对一个参数的多行解释，使用分号分割
2. 对应可选参数的解释，结尾不使用标点符号
3. 增加一些参数的默认值描述
4. 对常用参数的位置进行提前
5. 对必填参数不是使用注释
6. 对常用性能调优参数不使用注释
7. 增加一些参数在不同模式下的使用方式
8. 增加 routes 的描述
9. 更新一些参数描述的表达方式
10. 删除 TiKV Importer 配置参数中 compression-per-level 相关配置选项，让用户更加聚焦。

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -86,78 +91,91 @@ meta-schema-name = "lightning_metadata"
...
-batch-import-ratio = 0.75
+# 当后端模式是 "importer" 时，tikv-importer 的监听地址（需改为实际地址）。
+# addr = "172.16.31.10:8287"
+
+# 表路由
+[[routes]]
+schema-pattern = "test"
+table-pattern = "t2021*"
+target-schema = "test"
+target-table = "t"
```
Comment (sunzhaoyang) [inline tidb-lightning/tidb-lightning-configuration.md:155]: 这部分内容我刚删掉了。这是没拉最新的还是又加回来了？

Comment (ran-huang) [reply tidb-lightning/tidb-lightning-configuration.md:155]: 看起来是 @glkappe 加回来了

Comment (glkappe) [reply tidb-lightning/tidb-lightning-configuration.md:155]: @sunzhaoyang 
1. 截止到目前客户需要经常使用它
2. 我们目前没有方便的替代方案对于 routes
3. 他仅出现在其他文档中，但是在总的配置文件中却不存在，不合理
辛苦朝阳老师考虑下。

Comment (sunzhaoyang) [reply tidb-lightning/tidb-lightning-configuration.md:155]: route 现在很尴尬。和 mydumper.files 冲突，而且没有详细测试。因此用的时候用户容易踩坑，总的配置文件中我也已经移除了。

能否把客户经常使用的场景文档发我下？我瞅瞅mydumper.files能否满足，不能的话再排期把 route 功能 ga 掉之后再放到文档里（其实我也挺喜欢  route 的）

**Comment by ran-huang** [comment]
@glkappe could you please resolve the conflicts so we can proceed with reviewing and merging this PR?

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-6.6 label because the v6.6 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.6 and will no longer receive new updates.

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-7.0 label because the v7.0 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v7.0 and will no longer receive new updates.

**Comment by qiancai** [comment]
Removed the `needs-cherry-pick-release-7.2` label because the v7.2 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v7.2 and will no longer receive new updates.

**Pull request closed without merging**